whep_proxy/main.go

```diff
@@ -0,0 +1,532 @@
+package main
```

Comment:
I think this is largely fine - there is a discussion to be had on whether it is worth introducing an entirely new app into the codebase - but I digress.
What I will ask though: is with my above comments of running this in parallel with the python app, there should probably be a config option to disable this server if we don't need it. Though, Go is extremely lightweight and the only thing we're doing here is opening another port. I am generally in favor of keeping this here. I don't think it would add _that_ much added overhead that we really need to worry about it.

Reply:
Yea, this just starts a tiny web app in a fraction of a second. The actual heavy lifting happens when you start adding streams to the bridge.